test/conformance/conformance_global_control.cpp

```diff
@@ -230,6 +230,110 @@ TEST_CASE("terminate_on_exception: nested") {
     delete c0;
 }
 
+//! Testing setting the same value but different objects
+//! \brief \ref interface \ref error_guessing
+TEST_CASE("setting same value") {
+    const std::size_t value = 2;
+
+    oneapi::tbb::global_control* ctl1 = new oneapi::tbb::global_control(oneapi::tbb::global_control::max_allowed_parallelism, value);
+    oneapi::tbb::global_control* ctl2 = new oneapi::tbb::global_control(oneapi::tbb::global_control::max_allowed_parallelism, value);
+
+    std::size_t active = oneapi::tbb::global_control::active_value(oneapi::tbb::global_control::max_allowed_parallelism);
+    REQUIRE(active == value);
+    delete ctl2;
+
+    active = oneapi::tbb::global_control::active_value(oneapi::tbb::global_control::max_allowed_parallelism);
+    REQUIRE_MESSAGE(active == value, "Active value should not change, because of value duplication");
+    delete ctl1;
+}
+
+//! Testing lifetime control conformance
+//! \brief \ref interface \ref requirement
+TEST_CASE("prolong lifetime simple") {
```

Comment:
Does it make sense to move `task_scheduler_handle` test to separate conformance test ?

Reply:
It seems we need to fix the spec first because it states that `global_control.h` should be included

Reply:
here is the spec issue https://github.com/oneapi-src/oneAPI-spec/issues/398